app/src/main/java/com/github/gotify/service/WebSocketService.java

```diff
@@ -46,16 +62,17 @@
     private AtomicInteger lastReceivedMessage = new AtomicInteger(NOT_LOADED);
     private MissedMessageUtil missingMessageUtil;
 
+    private Picasso cache;
```

Comment:
While picasso does caching, it is not only a cache, so let's call it picasso.
```suggestion
    private Picasso picasso;
```